### PR TITLE
docs: add funnel and pyramid chart docs

### PR DESCRIPTION
**Pull request by pbenes**
JIRA: TNT-1432

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1266/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/855/ : SUCCESS in 6s

File: docs/10_vis__funnel_chart_component.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| drillableItems | false | [IDrillableItem[]](15_props__drillable_item.md) | An array of points and attribute values to be drillable |
+| ErrorComponent | false | Component | A component to be rendered if this component is in error state (see [ErrorComponent](15_props__error_component.md)) |
+| LoadingComponent | false | Component | A component to be rendered if this component is in loading state (see [LoadingComponent](15_props__loading_component.md)) |
+| onError | false | Function | A callback when the component updates its error state |
+| onExportReady | false | Function | A callback when the component is ready for exporting its data |
+| onLoadingChanged | false | Function | A callback when the component updates its loading state |
+| onDrill | false | Function | A callback when a drill is triggered on the component |
+
+The following example shows the supported `config` structure with sample values. For the descriptions of the individual options, see [ChartConfig](15_props__chart_config.md).
```
Comment (Mara3l) [inline docs/10_vis__funnel_chart_component.md:62]: ... For the `description` of the individual ...

File: docs/10_vis__funnel_chart_component.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: Funnel Chart
+sidebar_label: Funnel Chart
+copyright: (C) 2007-2023 GoodData Corporation
+id: funnel_chart_component
+---
+
+A **funnel chart** shows data as parts of a funnel. Funnel charts can be segmented by either multiple measures or an attribute.
```
Comment (Mara3l) [inline docs/10_vis__funnel_chart_component.md:8]: .. Funnel charts can be segmented by either multiple measures [ or a single attribute / or attributes ].

File: docs/10_vis__pyramid_chart_component.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+title: Pyramid Chart
+sidebar_label: Pyramid Chart
+copyright: (C) 2007-2023 GoodData Corporation
+id: pyramid_chart_component
+---
+
+A **pyramid chart** shows data as parts of a pyramid. Pyramid charts can be segmented by either multiple measures or an attribute.
```
Comment (Mara3l) [inline docs/10_vis__pyramid_chart_component.md:8]: Same as above

**Review by Mara3l — APPROVED** [review]
There are some changes, that I would personally change, but I leave it up to you to decide, whether you want to incorporate it. Other than that LGTM.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1267/ : SUCCESS in 1m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/856/ : SUCCESS in 4s

**Review by Mara3l — APPROVED** [review]
LGTM

**Review by xMort — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR589

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1268/ : SUCCESS in 1m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/180/ : SUCCESS in 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/375/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul/17/ : FAILURE in 2m 37s (non-voting)

**Pull request merged**